GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -8,6 +8,7 @@
             <th scope="col" class="govuk-table__header">Year</th>
             <th scope="col" class="govuk-table__header">Reporting requirement</th>
             <th scope="col" class="govuk-table__header">Report status</th>
+            <th scope="col" class="govuk-table__header"> </th>

```

Comment:
Is there a reason why we added a space character?

Reply:
Nope